apps/ewallet/config/dev.exs

```diff
@@ -1,9 +1,19 @@
 use Mix.Config
 
+endpoints = []
+
+endpoints =
```

Comment:
Can you put some explanation as comment about why we need to do `Code.ensure_compiled?/1` and the conditioning?

Reply:
Sure, adding this:

```
# We use 'Code.ensure_compiled?' to ensure the endpoints are loaded.
# When running tests from inside the Admin API/eWallet API/eWallet sub apps,
# those endpoint modules are not available and therefore should not be added to
# the list of endpoints that needs to be notified. If they are and the
# Event emitter tries to call them, it will result in a 500 error.
```

This could probably be improved tho, by maybe extracting it to some functions? I wasn't sure where to put them tho, so just went with the copy approach. Open to suggestions!